DeeployTest/generateNetwork_xdna2.py

```
@@ -0,0 +1,219 @@
+    graph = gs.import_onnx(onnx_graph)
+
+    inputs_npz = np.load(f'{args.dir}/inputs.npz')
+    outputs_npz = np.load(f'{args.dir}/outputs.npz')
+
+    test_inputs_f32 = [inputs_npz[x] for x in inputs_npz.files]
+    test_outputs_f32 = [outputs_npz[x] for x in outputs_npz.files]
+
+    inputTypes = {}
+    inputOffsets = {}
+
+    for index, (name, values) in enumerate(zip(inputs_npz.files, test_inputs_f32)):
```

Ruff (0.15.12)

[warning] Loop control variable `name` not used within loop body

Rename unused `name` to `_name`

(B007)